eng/common/pipelines/templates/steps/get-pr-owners.yml

```
@@ -18,12 +18,20 @@ steps:
         --kusto-database-var KUSTO_DB `
         --kusto-table-var KUSTO_TABLE `
         --identity "$(Build.QueuedBy)"
-      $resolvedIdentity = $result[-1] | ConvertFrom-Json
 
-      Write-Host $resolvedIdentity
+      $resolvedIdentity = ""
+      try { $resolvedIdentity = $result[-1] | ConvertFrom-Json } catch { $null}
```

```suggestion
      try { $resolvedIdentity = $result[-1] | ConvertFrom-Json } catch { }
```
No need to explicitly have a null here.